$DragonFly: src/gnu/usr.bin/binutils215/as/Attic/tc.h.patch,v 1.1 2005/05/08 00:35:48 corecode Exp $
===================================================================
RCS file: /space/cvs/dragonfly/src/contrib/binutils-2.15/gas/tc.h,v
retrieving revision 1.1
diff -u -r1.1 tc.h
--- tc.h	18 Dec 2004 20:21:20 -0000	1.1
+++ tc.h	6 May 2005 20:01:32 -0000
@@ -24,25 +24,6 @@
 
 extern const pseudo_typeS md_pseudo_table[];
 
-/* JF moved this here from as.h under the theory that nobody except MACHINE.c
-   and write.c care about it anyway.  */
-
-struct relax_type
-{
-  /* Forward reach. Signed number. > 0.  */
-  long rlx_forward;
-  /* Backward reach. Signed number. < 0.  */
-  long rlx_backward;
-
-  /* Bytes length of this address.  */
-  unsigned char rlx_length;
-
-  /* Next longer relax-state.  0 means there is no 'next' relax-state.  */
-  relax_substateT rlx_more;
-};
-
-typedef struct relax_type relax_typeS;
-
 extern const int md_reloc_size;	/* Size of a relocation record */
 
 char *md_atof (int what_statement_type, char *literalP, int *sizeP);
